src/Microsoft.AspNet.Mvc.TagHelpers/AnchorTagHelper.cs

```diff
@@ -25,7 +25,11 @@ public class AnchorTagHelper : TagHelper
         private const string RouteValuesPrefix = "asp-route-";
         private const string Href = "href";
 
-        [Activate]
+        public AnchorTagHelper(IHtmlGenerator generator)
+        {
+            Generator = generator;
```

Comment:
Should the properties that are now ctor injected by get-only? (That is, remove the `set;`.)

That's probably the common pattern for using ctor injection.